src/app/components/list/overview/index.html

```diff
@@ -75,6 +75,180 @@
     </sky-demo-page-property>
   </sky-demo-page-properties>
 
+  <sky-demo-page-properties sectionHeading="ListDataRequestModel properties">
+    <sky-demo-page-property
+      propertyName="filters"
+      isOptional="true"
+    >
+    Specifies a function to determine whether items are filtered. This property is required when using an in-memory data provider. This property accepts an array of <a stacheRouterLink="." fragment="listfiltermodel-properties"><sky-code>ListFilterModel</sky-code> objects</a>.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="isToolbarDisabled"
+      defaultValue="false"
+      isOptional="true"
+    >
+    Specifies whether the search bar and filter button are disabled. This property accepts <sky-code>boolean</sky-code> values.
```

Comment:
I'd change "Specifies" to "Indicates" because that's the tsandard wording for boolean properties. ... Also, I'd flip this around to use active voice: "Indicates whether to disable the search bar and filter button."